docs/developer-docs/jr-dev-rules.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Junior Developer Program Rules
+description: Consolidates scattered junior developer program rules, adds clarification and introduces some new rules
```

Comment:
This is what will appear in the Discord embed and on external search engines. Recommend something like:

```md
description: Comprehensive guidelines for the Junior Developer program, including rules, guidelines, and clarifications.
```

Reply:
Updated as recommended.  I appreciate the SEO tips.